module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -276,6 +276,17 @@ protected function getFakeCallNum()
         return $a . $b . "." . $c;
     }
 
+    /**
+     * Generate a fake call number prefix.
+     *
+     * @return string
+     */
+    protected function getFakeCallNumPrefix()
+    {
+        $codes = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+        return substr(str_shuffle($codes), 1, rand(0, 3));
+    }
```

Comment:
@demiankatz I'm not sure that this is very useful. I just did it to get started. If we had a small handful of predetermined prefixes that were randomly chosen, it would be easier to distinguish them from the rest of the call number. Would that be preferable?

Reply:
The test data doesn't need to be realistic, so maybe you could even do something like "Prefix" followed by a random digit.